source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,204 @@
+---
+id: add-new-attribute
+title: Add a new attribute
+---
+
+Let's say we want to add a new element to display on a product's page (for instance, a product's rate). You will have to:
+
+1. Add an attribute to your Magento 2.
+2. Expose the attribute in your GraphQL schema.
+3. Display it on screen.
+
+## Add a value to your database
+
+The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [How to Create Product Attributes in Magento 2](https://www.fastcomet.com/tutorials/magento2/product-attributes).
+
+## Expose the attribute in your GraphQL schema
+
+We want to display the `rate` of a product on the product page. To keep it simple, it will only consist in displaying a number, which can be manually edited by the merchant. This feature will be stored in a new module called `mymodule`.
+
+<blockquote class="info">
+You could directly edit the files at `node_modules/front-commerce/path/to/something`, but this is not recommended since running `npm install` might erase any modification.
+Learn [how to extend the GraphQL schema](/docs/essentials/extend-the-graphql-schema.html).
+</blockquote>
+
+```bash
+mkdir mymodule
+```
+
+In this example, the files we are going to override are located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`, so will create in our `mymodule` folder the following directories. More details [here](/docs/essentials/extend-the-graphql-schema.html).
```

Comment:
In this sentence, you talk about a React component and yet you're talking about the server in the following command.

I think that this sentence should be updated. The React Component is only relevant in the `How to find which files to edit` part.